src/daft-connect/src/lib.rs

```diff
@@ -49,10 +49,13 @@ impl ConnectionHandle {
     }
 }
 
-pub fn start(addr: &str) -> eyre::Result<ConnectionHandle> {
+pub fn start(addr: &str) -> eyre::Result<(ConnectionHandle, u16)> {
```

Comment:
sorry for the back and forth...  but I think we could probably stick the addr and port on `ConnectionHandle` instead of returning a tuple.